pkg/wallet/wallet.go

```diff
@@ -123,8 +130,34 @@ func (w *BaseWallet) Close() error {
 }
 
 // CreateDID returns new DID Document
-func (w *BaseWallet) CreateDID() (*did.Doc, error) {
-	return nil, fmt.Errorf("To be implemented")
+func (w *BaseWallet) CreateDID(method string) (*did.Doc, error) {
+	// Generate key pair
+	pub, err := w.CreateKey()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create DID: %w", err)
+	}
+
+	// DID identifier
+	id := fmt.Sprintf(didFormat, method, pub[:16])
+
+	// Supporting only one public key now
+	pubKey := did.PublicKey{
+		ID:         fmt.Sprintf(didPKID, id, 1),
+		Type:       "Curve25519", //TODO should be hardcoded?
```

Comment:
Where did you get this type? Also, we should not assume that wallet creates only one type of keys. One way to fix it  is to return key type from CreateKey. This can be done in separate PR.

Reply:
@fqutishat @llorllale  @sandrask @sudeshrshetty @rolsonquadras @Moopli  fyi ...

this Epic apparently mentions the 25519 key type value:
https://github.com/hyperledger/aries-framework-go/issues/96

also, this issue was added to crated a higher level Encrypter that will take in the key type and return a crypter for that specific type in the wallet: https://github.com/hyperledger/aries-framework-go/issues/273

Reply:
changed to `Ed25519VerificationKey2018` for now.
Once create keys in wallet starts supporting multiple keys, it should be dynamic.